src/Interpolation/CIC.hpp

```diff
@@ -63,7 +63,135 @@ namespace ippl {
             return (interpolationWeight<GatherPoint, Index>(wlo, whi) * ...)
                    * view(interpolationIndex<GatherPoint, Index>(args)...);
         }
+        template <unsigned long ScatterPoint, unsigned long... Index, typename T, unsigned Dim,
+                  typename IndexType>
+        KOKKOS_INLINE_FUNCTION constexpr int ZigzagScatterToPoint(
+            const std::index_sequence<Index...>&,
+            const typename ippl::detail::ViewType<ippl::Vector<T, 3>, Dim>::view_type& view,
+            const Vector<T, Dim>& wlo, const Vector<T, Dim>& whi,
+            const Vector<IndexType, Dim>& args, const Vector<T, Dim>& val, T scale, const Vector<T, Dim>& hr, const NDIndex<Dim> lDom, int nghost, const Vector<T, Dim>& source) {
+            (void)hr;
+            (void)lDom;
+            (void)nghost;
+            (void)source;
```

Comment:
Is this just to avoid warnings about unused arguments? Why are there unused arguments?